src/test/resources/org/prebid/server/it/floors/floors-test-auction-request-2.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Same as for request-1

Reply:
Cleared fields that are not needed from both auction requests.